Lib/concurrent/futures/_base.py

```diff
@@ -594,10 +600,23 @@ def map(self, fn, *iterables, timeout=None, chunksize=1):
                 before the given timeout.
             Exception: If fn(*args) raises for any values.
         """
+        if buffersize is not None and buffersize < 1:
+            raise ValueError("buffersize must be None or > 0.")
```

Comment:
For consistency, can you check if exception messages in the module have a period or not at the end, please?

Reply:
thanks @picnixz I have now removed the period ✅, turns out messages in this module don't have a period at the end... except the one I copied: `raise ValueError("chunksize must be >= 1.")` in `ProcessPoolExecutor.map`. 

Question: for the sake of consistency, one may be tempted to remove the period in `ValueError("chunksize must be >= 1.")`, but manipulating error messages is a dangerous non-backward compatible change that can mess with `assertRaisesRegex`-like tests, right?

Reply:
Let's be consistent with the all-but-one cases (namely no period) but let's not change legacy error messages. Indeed, it can break tests but it can also break users relying on exception messages although they shouldn't.

Reply:
@picnixz makes total sense ✅